### PR TITLE
[#142306] Update future scope and helper text

### PR DESCRIPTION
**Pull request by giladshanan**
# Release Notes

Updates a scope to be Oracle-friendly.  
Also makes the caption a bit broader and puts more detail info in hint text at the bottom of the page.

### CODE REVIEW

File: app/models/holiday.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Holiday < ApplicationRecord
 
   validates_presence_of :date
 
-  scope :future, -> { where("holidays.date >= ?", Time.current.to_date) }
```
Comment (rdunlop) [inline app/models/holiday.rb:7]: How hard would it be to add a spec to this which would show that it's failing on Oracle?

**Review by rdunlop — APPROVED** [review]

File: app/models/holiday.rb (hunk 1 above)
Comment (giladshanan) [reply app/models/holiday.rb:7]: I'm not sure how to do that (other than testing against Oracle).

Comment (rdunlop) [reply app/models/holiday.rb:7]: Oh, I forgot, I thought our CI was testing against oracle

Comment (giladshanan) [reply app/models/holiday.rb:7]: It does for nucore-nu, but not open

**Pull request merged**